File: FizzBuzz.js
function FizzBuzz(n){
    for (let i = 1; i < n; i++) {
        if (i % 15 == 0) console.log("FizzBuzz");
        else if (i % 3 == 0) console.log("Fizz");
        else if (i % 5 == 0) console.log("Buzz");
        else console.log(i);
    }
}

function main() {
   // const n = parseInt(readLine().trim(), 10);
    let n = 15;
    fizzBuzz(n);
}

File: fizzBuzz.c
int fizzBuzz(n){
    int i;
    for (i=0;i<n ; i++)
    {
    if (i %(3*5) == 0){
        printf("FizzBuzz\n");
        }
        else if ((i%3) == 0){
        printf("Fizz\n");
        }
        else if ((i%5) == 0){
        printf("Buzz\n");
        }
        else 
            printf("%d\n",i);
    }
}